src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs

```diff
@@ -206,7 +206,7 @@ public class InvitationsController : ControllerBase
 
         [Authorize(Roles = Permissions.IPO_READ)]
         [HttpGet("{id}/Attachments/{attachmentId}")]
-        public async Task<ActionResult<AttachmentDto>> GetAttachment(
+        public async Task<ActionResult<Query.GetAttachmentById.AttachmentDto>> GetAttachment(
```

Comment:
Undo line 209 + 223. Existing AttachmentDto in current namespace seems to be unused. Delete it